modules/effects/src/main/scala/dev/profunktor/redis4cats/redis.scala

```diff
@@ -528,6 +528,7 @@ private[redis4cats] class BaseRedis[F[_]: FutureLift: MonadThrow: Log, K, V](
     setArgs.ttl.foreach {
       case SetArg.Ttl.Px(d) => jSetArgs.px(d.toMillis)
       case SetArg.Ttl.Ex(d) => jSetArgs.ex(d.toSeconds)
+      case SetArg.Ttl.KeepTtl => jSetArgs.keepttl()
```

Comment:
Nitpick: I'd rename it to `Keep` since the `Ttl` is already part of the namespace (i.e. `SetArg.Ttl.Keep`)

Reply:
I see your point.
I think `KeepTtl` might be more informative in some cases, for example;
```scala
import dev.profunktor.redis4cats.effects.SetArg.Existence._
import dev.profunktor.redis4cats.effects.SetArg.Ttl._

redis.set("key", "value", SetArgs(Xx, KeepTtl)
```
It might be a niche case, but it's one of those cases where clutter in code is greatly reduced.

Reply:
It's not consistent with the current naming, though. `Px` and `Ex` by themselves don't say much, it's been named this way to use it with the full namespace --- i.e. `Ttl.Ex(d)`, `Ttl.Px(d)` and `Ttl.Keep`.

---

Still, users that don't like it can still use `Ex`, `Px` and `Keep` directly, though none of them say anything about TTL. 

Another option is to use qualified imports if that's a preference:

```scala
import dev.profunktor.redis4cats.effects.SetArg.Ttl.Keep as KeepTtl
```

Reply:
good point, changed.